tests/unit/StandaloneDirectorsFiling.spec.ts

```diff
@@ -1466,12 +1464,13 @@ describe('Change of Directors - BCOL error dialog on save', () => {
         ConfirmDialog: true,
         PaymentErrorDialog: true,
         ResumeErrorDialog: true,
+        BcolErrorDialog: true,
```

Comment:
As this dialog wasn't stubbed, the tests complained of being unable to find the component's ID tag to attach to.